News about OSS contributions (#229)
hms-dbmi/gehlenborglab-website: _news/2020-11-01-os-contributions.md

---
title: Open Source Software Contributions

members:
  - ilan-gold
  - trevor-manz
  - chuck-mccallum
 
blurb: Lab members contribute to multiple open source software projects.
---
Our work in the Gehlenborg lab depends on open source software, but we also give back to the community,
and are contributors to several projects.
- Ilan Gold [extended Deck.gl's tiling](https://github.com/visgl/deck.gl/pulls?q=is%3Apr+author%3Ailan-gold)
  to work for non-geographic tile sets,
  and Viv is now featured in the [Deck.gl showcase](https://deck.gl/showcase)
- Trevor Manz [presented](https://forum.image.sc/t/upcoming-calls-on-next-gen-bioimaging-data-tools-starting-oct-29/43489/34)
  work on Viv recently at an OME meeting on next-generation file formats (NGFF),
  and has made a [lot of PRs to zarr.js](https://github.com/gzuidhof/zarr.js/pulls?q=is%3Apr+author%3Amanzt).
- Open source isn't just about code. Chuck can be nit-picky about grammar, and at one point fixed

>   [dozens](https://github.com/cypress-io/cypress-documentation/pull/195/files) of it's/its mistakes in the cypress.io docs.

Reviewer: Not too trivial at all :)

Reviewer: Also deck.gl! https://github.com/visgl/deck.gl/pulls?q=is%3Apr+author%3Amccalluc

---
title: Open Source Software Contributions

members:
  - ilan-gold
  - trevor-manz
  - chuck-mccallum
 
blurb: Lab members contribute to multiple open source software projects.
---
Our work in the Gehlenborg lab depends on open source software, but we also give back to the community,
and are contributors to several projects.
- Ilan Gold [extended Deck.gl's tiling](https://github.com/visgl/deck.gl/pulls?q=is%3Apr+author%3Ailan-gold)
  to work for non-geographic tile sets,
  and Viv is now featured in the [Deck.gl showcase](https://deck.gl/showcase).
- Trevor Manz [presented](https://forum.image.sc/t/upcoming-calls-on-next-gen-bioimaging-data-tools-starting-oct-29/43489/34)
  work on Viv recently at an OME meeting on next-generation file formats (NGFF),
  and has made a [lot of PRs to zarr.js](https://github.com/gzuidhof/zarr.js/pulls?q=is%3Apr+author%3Amanzt).

> - Open source isn't just about code. Chuck can be nit-picky about grammar,

Reviewer 2: Can we change that to "Open source isn't just about code. Chuck McCallum has contributed to the documentation of [Cypress.io](https://github.com/cypress-io/cypress-documentation/pull/195/files) and [Deck.gl](https://github.com/visgl/deck.gl/pulls?q=is%3Apr+author%3Amccalluc)".

> - Open source isn't just about code. Chuck can be nit-picky about grammar,
>   and has fixed [it's/its mistakes](https://github.com/cypress-io/cypress-documentation/pull/195/files)
>   in the cypress.io docs and 
>   [typos](https://github.com/visgl/deck.gl/pulls?q=is%3Apr+author%3Amccalluc) in the deck.gl docs.

Author: ```suggestion
- Open source isn't just about code. Chuck McCallum has contributed to the documentation of
  [Cypress.io](https://github.com/cypress-io/cypress-documentation/pull/195/files)
  and [Deck.gl](https://github.com/visgl/deck.gl/pulls?q=is%3Apr+author%3Amccalluc).
```
